.ci/load/scripts/locustfile.py

```diff
@@ -0,0 +1,13 @@
+from locust import HttpUser, task, constant
+
+
+class QuickstartUser(HttpUser):
+    wait_time = constant(1)
```

Comment:
IIUC, `constant` is still vulnerable to coordinated omission as it waits one second after the requests has been completed. By using `constant_pacing`, we can ensure a constant RPS, regardless of how long it takes to serve the request.

An example of when this can be problematic: Imagine there's a 1-minute stop-the-world JVM pause due to a major garbage collection. During that time, we'd effectively "look away" by waiting for the current requests to execute before executing others. Only by executing pre-defined request/sec rate, no matter how the system under test responds, we can fully include the effect of the long stop-the-world pause in the test results.

Reply:
Ah, OK. I'll change this to `constant_pacing`.